adhocracy4/comments_async/static/comments_async/comment.jsx

```diff
@@ -377,39 +377,40 @@ export default class Comment extends React.Component {
               lastEdit={this.state.moderatorFeedback.last_edit}
               feedbackText={this.state.moderatorFeedback.feedback_text}
             />}
+          <div className="a4-comments__action">
+            <div className="row">
```

Comment:
this looks scary but i just added a wrapper div arounf row so it has same width as elements above